golangci.yml

```diff
@@ -127,13 +128,15 @@ issues:
         - nosnakecase
         - noctx
         - goconst
+        - err113
 
     # Exclude lll issues for long lines with go:generate
     - linters:
         - lll
       source: "^//go:generate "
     - linters:
         - goerr113
+        - err113
```

Comment:
you only need to use `err113` here

Reply:
yes thanks I fixed it in https://github.com/fortio/workflows/pull/36/commits/05045f6e545d4fcbddedfcbff6327b25b9a165b4